src/pages/homeCollection/kitAssembly.js

```diff
@@ -142,16 +143,16 @@ const processAssembledKit = () => {
         const confirmScannedBarcodeValue = document.getElementById('scannedBarcode2')?.value?.trim();
 
         const querySupplyKitIdValue = document.getElementById('supplyKitId').value.trim();
-        const supplyKitIdValue = (querySupplyKitIdValue !== undefined) ? querySupplyKitIdValue: 0;
+        const supplyKitIdValue = (querySupplyKitIdValue !== undefined) ? querySupplyKitIdValue.toUpperCase(): 0;
 
         const queryReturnKitIdValue = document.getElementById('returnKitId')?.value?.trim();
-        const returnKitIdValue = (queryReturnKitIdValue !== undefined) ? queryReturnKitIdValue : 0;
+        const returnKitIdValue = (queryReturnKitIdValue !== undefined) ? queryReturnKitIdValue.toUpperCase() : 0;
 
         const queryCollectionCupIdValue = document.getElementById('cupId')?.value?.trim();
-        const collectionCupIdValue = (queryCollectionCupIdValue !== undefined) ? queryCollectionCupIdValue : 0;
+        const collectionCupIdValue = (queryCollectionCupIdValue !== undefined) ? queryCollectionCupIdValue.toUpperCase() : 0;
 
         const queryCollectionCardIdValue = document.getElementById('cardId')?.value?.trim();
-        const collectionCardIdValue = (queryCollectionCardIdValue !== undefined) ? queryCollectionCardIdValue : 0;
+        const collectionCardIdValue = (queryCollectionCardIdValue !== undefined) ? queryCollectionCardIdValue.toUpperCase() : 0;
```

Comment:
Will it be better to use string ("0"), instead of number (0), for type consistency?